src/graphql/queries.js

```diff
@@ -17,75 +17,155 @@
 
 // Code related to GraphQL queries, fragments, variables, etc.
 
+import gql from 'graphql-tag'
+// eslint-disable-next-line no-unused-vars
+import { DocumentNode } from 'graphql'
+
 // IMPORTANT: queries here may be used in the offline mode to create mock data. Before removing or renaming
 // queries here, please check under the services/mock folder for occurrences of the variable name.
 
 /**
- * Query used to retrieve data for a workflow Tree view.
- * @type {string}
+ * @type {DocumentNode}
  */
-export const WORKFLOW_TREE_QUERY = `
-subscription {
-  workflows(ids: ["WORKFLOW_ID"]) {
-    id
-    name
-    status
-    owner
-    host
-    port
-    cyclePoints: familyProxies(ids: ["root"]) {
+const WORKFLOW_TREE_DELTAS_SUBSCRIPTION = gql`
+subscription OnWorkflowDeltasData ($workflowId: ID) {
+  deltas (workflows: [$workflowId], stripNull: true) {
+   ...WorkflowTreeDeltas
+  }
+}
+
+# TREE DELTAS BEGIN
+
+fragment WorkflowTreeDeltas on Deltas {
+  id
+  shutdown
+  added {
+    ...WorkflowTreeAddedData
+  }
+  updated {
+    ...WorkflowTreeUpdatedData
```

Comment:
I think the added and updated fragments should always be identical for any rational subscription.

Reply:
The updated fragment is a subset of the added fragment now, i.e. the only simplification that we could have I think is using the updated in the added fragment, but I think that could be confusing.

The reason for that is that the added data contains the initial burst of data. And the added data contains cycle points, while cycle points are not in the updated as I think a cycle point is not updated?